crates/swc_bundler/src/bundler/optimize.rs

```diff
@@ -28,6 +28,7 @@ where
                         top_retain: Default::default(),
                     },
                     self.unresolved_mark,
+                    false,
```

Comment:
`swc_bundler` and `swc_ecma_minifier` use `dce` function, so I set false in this time. Should the user be able to set this value to arbitray value? I think this probably should, but I don't understand how to implement it in details so I would appreciate any advice you could give me.

Reply:
I think it does not matter, because `swc_bundler` is not actively maintained in favor of turbopack